### PR TITLE
Run tests for #7204

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by ArturGajowy** [comment]
@electrum Thx for looking into it! I've had a look at the failures and it seems that the modernizer violation ('prefer hashmap over hashtable' in JndiUtils) is unrelated and probably can only be solved by excluding either the rule or the file in quesiton (using `ignorePackages`).

The product test failure seems intermittent, I restarted the jobs.

Let me know if I can help :)

**Comment by electrum** [comment]
I moved the `ignorePackage` item to the one in `presto-main` and that fixes it

File: src/modernizer/violations.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!--
```
Comment (electrum) [inline src/modernizer/violations.xml]: Let's remove the license header (we don't do that for other XML files which are configuration) and format this with four spaces (like other XML files).

**Comment by electrum** [comment]
I'm rebasing this and running the tests, so don't worry about making changes now

**Pull request closed without merging**